mergify_engine/actions/rebase.py

```diff
@@ -59,6 +60,19 @@ def run(self, ctxt: context.Context, rule: rules.EvaluatedRule) -> check_api.Res
             if output:
                 return output
 
+            repo_full_name = ctxt.pull["base"]["repo"]["full_name"]
+            repo_info = ctxt.client.item(f"/repos/{repo_full_name}")
```

Comment:
Not a big deal, but I think this would be slightly more accurate:
```suggestion
            repo_info = ctxt.client.item(f"{ctxt.pull["base"]["repo"]["url"]")
```